cmd/main.go

```diff
@@ -153,7 +159,7 @@ func main() {
 		OADPNamespace:      oadpNamespace,
 		EnforcedBackupSpec: enforcedBackupSpec,
 	}).SetupWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create controller", "controller", "NonAdminBackup")
+		setupLog.Error(err, unableToCreateControllerString, controllerString, "NonAdminBackup")
```

Comment:
reframe this to `unable to setup <controller-name> controller with manager`

Reply:
Fixed, will commit soon. Removed constant controller and controllerString as well, because those are now static strings.

Reply:
Can we close this as resolved?